### ISSUE TITLE
build fails on atk (VS 2017)

### ISSUE THREAD

**Issue by tjwilli58** [wingtk/gvsbuild#264]
I having problems building when I get to building atk. This seems to be a similar problem to #260 .
I got gettext built, but moving on, I get 

`..\..\Anaconda3\python.exe build.py build pygobject --vs-ver=15 --platform=x64  --vs-install-path="%vsinstalldir%\"
`
```
Building project atk
ninja: no work to do.
[0/1] Running all tests.
SUCCESS: The process with PID 128852 (child process of PID 128532) has been terminated.
SUCCESS: The process with PID 128092 (child process of PID 128532) has been terminated.
SUCCESS: The process with PID 126392 (child process of PID 128532) has been terminated.
SUCCESS: The process with PID 128348 (child process of PID 128532) has been terminated.
1/5 testdocument                            TIMEOUT 30.64 s
2/5 testrole                                TIMEOUT 30.65 s
3/5 testrelation                            TIMEOUT 30.64 s
4/5 teststateset                            TIMEOUT 30.64 s
SUCCESS: The process with PID 128172 (child process of PID 128532) has been terminated.
5/5 testvalue                               TIMEOUT 30.30 s

OK:         0
FAIL:       5
SKIP:       0
TIMEOUT:    5

```

> During the tests, I get a popup window saying
> The procedure entry point iconv_open could not be located in the dynamic link library C:\gtk-build\gtk\x64\release\bin\intl.dll.
> 

How do I get past this?
Thanks for any help.

**Comment by guruDanny67** [comment]
It seems more a problem of wrong dll been load / created (intl.dll, from iconv I think).

Try to build, once, with --from-scratch to start completely clean and drop any wrong file laying around.

If this doesn't work try to launch the build with the -d option (without the --from-scratch) and try to post the output of the command to see if there's something.

Another change is to compare the export from the dll built in x64 mode and the one built in x86 mode.

Hope this helps

**Comment by tjwilli58** [comment]
Thanks.  I'll give that a try, although I did delete the entire build directory once trying things out.
I did get atk to build on x86. (Problems building other pkgs kater on, but one thing at a time)

**Comment by tjwilli58** [comment]
build --from-scratch still gave errors for atk build.  Here's the output from -d option:

```
Building project atk
Debug: directory C:\gtk-build\build\x64\release\atk already exists
Debug: running ninja, cwd=C:\gtk-build\build\x64\release\atk-meson, path+=None
ninja: no work to do.
Debug: running ninja test, cwd=C:\gtk-build\build\x64\release\atk-meson, path+=None
[0/1] Running all tests.
1/5 testdocument                            FAIL     7.36 s
2/5 testrole                                FAIL     7.36 s
3/5 testrelation                            FAIL     7.36 s
4/5 teststateset                            FAIL     7.36 s
5/5 testvalue                               FAIL     1.46 s

OK:         0
FAIL:       5
SKIP:       0
TIMEOUT:    0


The output from the failed tests:

1/5 testdocument                            FAIL     7.36 s

--- command ---
G_TEST_SRCDIR='C:\gtk-build\build\x64\release\atk\tests' G_TEST_BUILDDIR='C:\gtk-build\build\x64\release\atk-meson\tests' C:\gtk-build\build\x64\release\atk-meson\tests\testdocument.exe
-------

2/5 testrole                                FAIL     7.36 s

--- command ---
G_TEST_SRCDIR='C:\gtk-build\build\x64\release\atk\tests' G_TEST_BUILDDIR='C:\gtk-build\build\x64\release\atk-meson\tests' C:\gtk-build\build\x64\release\atk-meson\tests\testrole.exe
-------

3/5 testrelation                            FAIL     7.36 s

--- command ---
G_TEST_SRCDIR='C:\gtk-build\build\x64\release\atk\tests' G_TEST_BUILDDIR='C:\gtk-build\build\x64\release\atk-meson\tests' C:\gtk-build\build\x64\release\atk-meson\tests\testrelation.exe
-------

4/5 teststateset                            FAIL     7.36 s

--- command ---
G_TEST_SRCDIR='C:\gtk-build\build\x64\release\atk\tests' G_TEST_BUILDDIR='C:\gtk-build\build\x64\release\atk-meson\tests' C:\gtk-build\build\x64\release\atk-meson\tests\teststateset.exe
-------

5/5 testvalue                               FAIL     1.46 s

--- command ---
G_TEST_SRCDIR='C:\gtk-build\build\x64\release\atk\tests' G_TEST_BUILDDIR='C:\gtk-build\build\x64\release\atk-meson\tests' C:\gtk-build\build\x64\release\atk-meson\tests\testvalue.exe
-------

Full log written to C:\gtk-build\build\x64\release\atk-meson\meson-logs\testlog.txt
FAILED: meson-test
"C:\\Users\\Tim\\Anaconda3\\python.exe" "-u" "C:\\gtk-build\\tools\\meson-0.44.0\\meson.py" "test" "--no-rebuild" "--print-errorlogs"
ninja: build stopped: subcommand failed.
Traceback (most recent call last):
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\utils\builder.py", line 325, in build
    self.__build_one(p)
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\utils\builder.py", line 404, in __build_one
    proj.build()
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\projects.py", line 66, in build
    Meson.build(self, meson_params='-Ddisable_introspection=true -Denable_docs=false', make_tests=True)
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\utils\base_builders.py", line 58, in build
    self.builder.exec_vs('ninja test', working_dir=ninja_build)
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\utils\builder.py", line 614, in exec_vs
    self.__execute(self.__sub_vars(cmd), working_dir=working_dir, add_path=add_path, env=self.vs_env)
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\utils\builder.py", line 646, in __execute
    subprocess.check_call(args, cwd=working_dir, env=env, shell=True)
  File "C:\Users\Tim\Anaconda3\lib\subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'ninja test' returned non-zero exit status 1.
Error: atk build failed

```
I've seen other posts with `--win-sdk-ver WIN_SDK_VER` used. Would that make a difference?

**Comment by guruDanny67** [comment]
There is something strange: the first 'ninja:no work to do' means that the
script already built (or try) the atk library.

One test to do could be to launch the script dev-shell (use dev-shell as
the project name): you end up in the build dir, then:

cd atk-.meson
ninja -t clean
ninja

This should clean up the ninja build file and rebuild the library from zero.


I don't have any more idea, sorry :(


2018-06-19 3:57 GMT+02:00 Tim Williams <notifications@github.com>:

**Comment by tjwilli58** [comment]
Pardon my ignorance, but where is this 'dev-shell'?  All I know about is running the build.py script

**Comment by guruDanny67** [comment]
Using dev-shell as a project (e.g. python build.py --vs-ver 15 ...
dev-shell) you are in a command prompt with all the tools enabled: it's
used to test the build of new projects or to see what's happening when
something goes wrong.

We really need some developer docs :)

2018-06-19 22:34 GMT+02:00 Tim Williams <notifications@github.com>:

**Comment by tjwilli58** [comment]
At work now, so I don't have my home laptop where I have gvsbuild, but I don't remember seeing `dev-shell` on the list of projects when I did a `build.py list`

**Comment by guruDanny67** [comment]
Try it, if you're using an updated version of the script is present.

It's a bug I realize now (thanks, btw) I put in one of the last commit,
when I want to avoid that making all the project build also this(so i can
launch a series of builds, one aftaer the other, on an unattended PC=

2018-06-20 18:08 GMT+02:00 Tim Williams <notifications@github.com>:

**Comment by tjwilli58** [comment]
On bus home now. I'll do a fetch and try it tonight. Thanks.

**Comment by tjwilli58** [comment]
@guruDanny67 , I got a fresh download, tried to build atk again (failed), then did a build of dev-shell.
After following your instructions, I tried to build atk again. Same result, but after looking at `ninja -t targets`, I did a `ninja test`, (failed as before), and then did a `ninja install`.

That may not be a good idea, but I'll see if I can go further in building pygobject.

**Comment by tjwilli58** [comment]
Failed on atk test again while building cairo

**Comment by guruDanny67** [comment]
I build the x64 version of all the gtk3 stack without problems. The intl.dll is 88.576 bytes (and the .pdb 815.104) while the iconv.dll is 69'.928 bytes. 

if it's (almost) the same size you can try, in the dev-shell, to add the build stack bin dir at the beginning of the path:

set PATH=C:\gtk-build\gtk\x64\release\bin\intl.dll;%PATH%

Then go under atk-meson and launch ninja test (uncommentting them, of course).

If they ends ok you have some iconv.dll in the path before the gtk bin dir (that we add before configuring the vs paths).

Also, try to post your complete path in the dev-shell and look in each of the dir if there is a iconv.dll or intl.dll

**Comment by tjwilli58** [comment]
Success! (Well, almost) 
It turned out  that iconv.dll is also in Anaconda\Library\bin. I did the 

> set PATH=C:\gtk-build\gtk\x64\release\bin\intl.dll;%PATH%
but ninja test still failed. I then finally saw the value of PATH after a `ninja clean`, `ninja`, `ninja test`.

`PATH='C:\Users\Tim\Anaconda3\Library\bin;C:\gtk-build\gtk\x64\release\bin\intl.dll;C:\gtk-build\tools\yasm;C:\Users\Tim\Anaconda3;C:\gtk-build\tools\perl\x64\bin;C:\gtk-build\tools\ninja;C:\gtk-build\tools\nasm-2.13.03;C:\gtk-build\tools\go\go\bin;C:\gtk-build\tools\cmake-3.7.2-win64-x64\bin;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\bin\HostX64\x64;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCPackages;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions\Microsoft\TestWindow;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\bin\Roslyn;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Team Tools\Performance Tools\x64;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Team Tools\Performance Tools;C:\Program Files (x86)\Microsoft Visual Studio\Shared\Common\VSPerfCollectionTools\\x64;C:\Program Files (x86)\Microsoft Visual Studio\Shared\Common\VSPerfCollectionTools\;C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.1 Tools\x64\;C:\Program Files (x86)\HTML Help Workshop;C:\Program Files (x86)\Windows Kits\10\bin\10.0.17134.0\x64;C:\Program Files (x86)\Windows Kits\10\bin\x64;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\\MSBuild\15.0\bin;C:\WINDOWS\Microsoft.NET\Framework64\v4.0.30319;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\;C:\gtk-build\gtk\x64\release\bin;c:\windows\microsoft.net\framework\v4.0.30319;c:\windows\system32;c:\windows;c:\windows\system32\wbem;c:\windows\system32\windowspowershell\v1.0;c:\windows\system32\openssh;d:\program files\git\cmd;C:\Msys64\usr\bin;C:\Users\Tim\Anaconda3\lib\site-packages\pypiwin32_system32' 

I renamed  iconv.dll and successfully built atk, gdk-pixbuf. I thought this could be an issue after seeing a post on [StackOverflow](https://stackoverflow.com/questions/7262362/mingw-the-procedure-entry-point-libiconv-could-not-be-located). 

I didn't know where to look for where this might be I thought it should be somewhere under VS2017, but doing a search from `C:\Program Files (x86)` didn't find anything.

gobject-inspection build failed though. I'm going to try and rebuild from scratch and see what happens.

**Comment by tjwilli58** [comment]
Decided to download Miniconda3. Builds flawlessly. (at least up to gobject-introspection)
I new uninstalled libiconv from anaconda3  and will try a build. (build finished as I was writing this.)

Getting late. I have to work in the morning, but it looks like the problem lies in libiconv in Anaconda. 

I installed `Anaconda3-5.0.1-Windows-x86_64.exe`, so an update may also be in order.

**Comment by tjwilli58** [comment]
I can build gtk3-full fully with Miniconda. Still can't get past gobject-introspection

```
Generating PangoCairo-1.0.gir...
        C:\Users\Tim\Anaconda3\python.exe C:\gtk-build\gtk\x64\release\bin\g-ir-scanner  --verbose -no-libtool   --namespace=PangoCairo  --nsversion=1.0         --pkg=gobject-2.0 --pkg=cairo   --library=pango-1.0 --library=pangocairo-1.0
  --add-include-path=C:\gtk-build\gtk\x64\release\share\gir-1.0  --include=GObject-2.0 --include=cairo-1.0 --include=win32-1.0   --pkg-export=pangocairo          --cflags-begin         -I.. -DG_LOG_DOMAIN="Pango" -DPANGO_ENABLE_BACKEND -DPANGO_ENABLE_ENGINE        --cflags-end    --include-uninstalled=./Pango-1.0.gir --c-include pango/pangocairo.h    --filelist=PangoCairo_1_0_gir_list      -o PangoCairo-1.0.gir
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\bin\HostX64\x64\cl.exe -E -P -C -DG_LOG_DOMAIN=Pango -DPANGO_ENABLE_BACKEND -DPANGO_ENABLE_ENGINE -D_USE_DECLSPECS_FOR_SAL -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_WARNINGS -DSAL_NO_ATTRIBUTE_DECLARATIONS -I. -IC:\gtk-build\build\x64\release\pango -IC:\include\cairo -IC:\gtk-build\gtk\x64\release\include\glib-2.0 -IC:\gtk-build\gtk\x64\release\lib\glib-2.0\include C:\gtk-build\build\x64\release\pango\win32\g-ir-cpp-7svuq4ba.c
Microsoft (R) C/C++ Optimizing Compiler Version 19.14.26430 for x64
Copyright (C) Microsoft Corporation.  All rights reserved.

g-ir-cpp-7svuq4ba.c
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\include\vcruntime.h:83: syntax e' at '__pragma'd identifier in '__pragma(pack(push, 8))
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\include\vcruntime.h:296: syntax ' at '__pragma'ed identifier in '__pragma(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt.h:1982: syntax error, unexpected identifier in ' at '__pragma'pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\stddef.h:57: syntax error, unexpected identifier in '__' at '__pragma'))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\float.h:402: syntax error, unexpected identifier in '__' at '__pragma'))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_wtime.h:13: syntax error, unexpected identifier' at '__pragma'ack(push, 8))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_wtime.h:208: syntax error, unexpected identifie' at '__pragma'pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\time.h:601: syntax error, unexpected identifier in '__p' at '__pragma')
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_malloc.h:13: syntax error, unexpected identifie' at '__pragma'pack(push, 8))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_malloc.h:228: syntax error, unexpected identifi' at '__pragma'(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\malloc.h:179: syntax error, unexpected identifier in '_' at '__pragma'p))
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\include\stdarg.h:13: syntax erro' at '__pragma'dentifier in '__pragma(pack(push, 8))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\signal.h:14: syntax error, unexpected identifier in '__' at '__pragma'h, 8))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\signal.h:65: syntax error, unexpected identifier in '__' at '__pragma'))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\errno.h:14: syntax error, unexpected identifier in '__p' at '__pragma', 8))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\errno.h:130: syntax error, unexpected identifier in '__' at '__pragma'))
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\include\vcruntime_string.h:106: ' at '__pragma'nexpected identifier in '__pragma(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_memcpy_s.h:88: syntax error, unexpected identif' at '__pragma'a(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_memory.h:114: syntax error, unexpected identifi' at '__pragma'(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_wstring.h:647: syntax error, unexpected identif' at '__pragma'a(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\string.h:592: syntax error, unexpected identifier in '_' at '__pragma'p))
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\include\excpt.h:14: syntax error' at '__pragma'entifier in '__pragma(pack(push, 8))
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\include\excpt.h:82: syntax error' at '__pragma'entifier in '__pragma(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_wctype.h:13: syntax error, unexpected identifie' at '__pragma'pack(push, 8))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_wctype.h:199: syntax error, unexpected identifi' at '__pragma'(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\ctype.h:234: syntax error, unexpected identifier in '#l' at '__pragma'ogram Files (x86)\\Windows Kits\\10\\include\\10.0.17134.0\\ucrt\\ctype.h"
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\basetsd.h:301: syntax error, unexpected identifier, e' at 'p' ')' or ',' in '    const void * __ptr32 p
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\basetsd.h:310: syntax error, unexpected identifier, e' at 'h' ')' or ',' in '    const void * __ptr32 h
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\basetsd.h:318: syntax error, unexpected identifier, e' at 'PtrToPtr32';' in 'PtrToPtr32(
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\basetsd.h:320: syntax error, unexpected ')', expectin' at ')' ';' in '    )
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\basetsd.h:322: syntax error, unexpected ')', expectin' at ')' ';' in '    return((void * __ptr32) (unsigned long) (ULONG_PTR) p);
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:390: syntax error, unexpected identifier, expecti' at 'PVOID64'in 'typedef void * __ptr64 PVOID64;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:456: syntax error, unexpected '*', expecting ',' ' at '*'n 'typedef   WCHAR __unaligned *LPUWSTR, *PUWSTR;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:460: syntax error, unexpected '*', expecting ',' ' at '*'n 'typedef   const WCHAR __unaligned *LPCUWSTR, *PCUWSTR;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:464: syntax error, unexpected '*', expecting ',' ' at '*'n 'typedef   WCHAR __unaligned *PUZZWSTR;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:465: syntax error, unexpected '*', expecting ',' ' at '*'n 'typedef   const WCHAR __unaligned *PCUZZWSTR;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:469: syntax error, unexpected '*', expecting ',' ' at '*'n 'typedef  WCHAR __unaligned *PUNZWCH;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:470: syntax error, unexpected '*', expecting ',' ' at '*'n 'typedef  const WCHAR __unaligned *PCUNZWCH;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:2526: syntax error, unexpected INTEGER in 'typede' at '16'__declspec(align(16)) _M128A {
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:2529: syntax error, unexpected '}' in '} M128A, *' at '}'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:2535: syntax error, unexpected INTEGER in 'typede' at '16'__declspec(align(16)) _XSAVE_FORMAT {
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:2549: syntax error, unexpected identifier in '   ' at 'M128A'Registers[8];
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:2563: syntax error, unexpected '}' in '} XSAVE_FO' at '}'XSAVE_FORMAT;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:2567: syntax error, unexpected INTEGER in 'typede' at '8' __declspec(align(8)) _XSAVE_AREA_HEADER {
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:2571: syntax error, unexpected '}' in '} XSAVE_AR' at '}'R, *PXSAVE_AREA_HEADER;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:2573: syntax error, unexpected INTEGER in 'typede' at '16'__declspec(align(16)) _XSAVE_AREA {
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:2582: syntax error, unexpected identifier in '   ' at 'PXSAVE_AREA'a;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:2594: syntax error, unexpected '}' in '} XSTATE_C' at '}'*PXSTATE_CONTEXT;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:2965: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:3009: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:3332: syntax error, unexpected identifier, expect' at '__getcallerseflags'allerseflags (
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:3544: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:3582: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:3787: syntax error, unexpected identifier, expect' at 'XMM_SAVE_AREA32'edef XSAVE_FORMAT XMM_SAVE_AREA32, *PXMM_SAVE_AREA32;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:3824: syntax error, unexpected INTEGER in 'typede' at '16'__declspec(align(16)) _CONTEXT {
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:3902: syntax error, unexpected identifier in '   ' at 'XMM_SAVE_AREA32'ltSave;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:3941: syntax error, unexpected '}' in '} CONTEXT,' at '}'XT;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:4023: syntax error, unexpected identifier in '   ' at 'PCONTEXT'xtRecord;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:4029: syntax error, unexpected '}' in '} DISPATCH' at '}'XT, *PDISPATCHER_CONTEXT;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:4057: syntax error, unexpected identifier in '   ' at 'PM128A'loatingContext[16];
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:4100: syntax error, unexpected '}' in '} KNONVOLA' at '}'TEXT_POINTERS, *PKNONVOLATILE_CONTEXT_POINTERS;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7670: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7684: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7698: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7711: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7724: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7738: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7752: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7765: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7778: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7792: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7806: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7819: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7832: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7846: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7860: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7873: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7896: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7910: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7923: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7937: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7950: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7964: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7977: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:7991: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8008: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8019: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8030: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8041: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8052: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8063: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8076: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8089: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8102: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8115: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8128: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8139: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8150: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8161: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8174: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8187: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8200: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8211: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8222: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8233: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8246: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8259: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8272: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8283: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8294: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8305: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8318: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8331: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8456: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8467: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8478: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8489: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8502: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8515: syntax error, unexpected identifier in '__f' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:8798: syntax error, unexpected identifier in '   ' at 'PCONTEXT'xtRecord;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:12623: syntax error, unexpected INTEGER in 'typed' at '16' __declspec(align(16)) _MEMORY_BASIC_INFORMATION64 {
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:12633: syntax error, unexpected '}' in '} MEMORY_' at '}'FORMATION64, *PMEMORY_BASIC_INFORMATION64;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:12747: syntax error, unexpected INTEGER in 'typed' at '8't __declspec(align(8)) MEM_EXTENDED_PARAMETER {
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:12762: syntax error, unexpected '}' in '} MEM_EXT' at '}'RAMETER, *PMEM_EXTENDED_PARAMETER;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:12995: syntax error, unexpected identifier in '  ' at 'PVOID64'er;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:12997: syntax error, unexpected '}' in '}FILE_SEG' at '}'MENT, *PFILE_SEGMENT_ELEMENT;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:16778: syntax error, unexpected '*', expecting ',' at '*' in 'typedef IMAGE_SYMBOL __unaligned *PIMAGE_SYMBOL;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:16797: syntax error, unexpected '*', expecting ',' at '*' in 'typedef IMAGE_SYMBOL_EX __unaligned *PIMAGE_SYMBOL_EX;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:16928: syntax error, unexpected '*', expecting ',' at '*' in 'typedef IMAGE_AUX_SYMBOL_TOKEN_DEF __unaligned *PIMAGE_AUX_SYMBOL_TOKEN_DEF;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:16976: syntax error, unexpected '*', expecting ',' at '*' in 'typedef IMAGE_AUX_SYMBOL __unaligned *PIMAGE_AUX_SYMBOL;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:17007: syntax error, unexpected '*', expecting ',' at '*' in 'typedef IMAGE_AUX_SYMBOL_EX __unaligned *PIMAGE_AUX_SYMBOL_EX;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:17043: syntax error, unexpected '*', expecting ',' at '*' in 'typedef IMAGE_RELOCATION __unaligned *PIMAGE_RELOCATION;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:17444: syntax error, unexpected '*', expecting ',' at '*' in 'typedef IMAGE_LINENUMBER __unaligned *PIMAGE_LINENUMBER;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:17459: syntax error, unexpected '*', expecting ',' at '*' in 'typedef IMAGE_BASE_RELOCATION __unaligned * PIMAGE_BASE_RELOCATION;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:17657: syntax error, unexpected '*', expecting ',' at '*' in 'typedef IMAGE_IMPORT_DESCRIPTOR __unaligned *PIMAGE_IMPORT_DESCRIPTOR;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:17886: syntax error, unexpected '*', expecting ',' at '*' in 'typedef IMAGE_PROLOGUE_DYNAMIC_RELOCATION_HEADER __unaligned * PIMAGE_PROLOGUE_DYNAMIC_RELOCATION_HEADER;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:17896: syntax error, unexpected '*', expecting ',' at '*' in 'typedef IMAGE_EPILOGUE_DYNAMIC_RELOCATION_HEADER __unaligned * PIMAGE_EPILOGUE_DYNAMIC_RELOCATION_HEADER;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:18604: syntax error, unexpected identifier, expec' at 'ContextRecord'      PCONTEXT ContextRecord
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:18727: syntax error, unexpected identifier, expec' at 'ContextRecord'      PCONTEXT ContextRecord,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:18729: syntax error, unexpected ')', expecting ',' at ')' in '    );
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:18746: syntax error, unexpected identifier in '  ' at 'PCONTEXT'ntextRecord,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:18765: syntax error, unexpected identifier in '  ' at 'PCONTEXT'ntextRecord,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:18767: syntax error, unexpected typedef-name, exp' at 'PDWORD64'er or '(' or '*' in '      PDWORD64 EstablisherFrame,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:18768: syntax error, unexpected identifier, expec' at 'ContextPointers'    PKNONVOLATILE_CONTEXT_POINTERS ContextPointers
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:19175: syntax error, unexpected INTEGER in 'typed' at '16' __declspec(align(16)) _SLIST_ENTRY {
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:19177: syntax error, unexpected '}' in '} SLIST_E' at '}'SLIST_ENTRY;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:19189: syntax error, unexpected INTEGER in 'typed' at '16'__declspec(align(16)) _SLIST_HEADER {
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:19200: syntax error, unexpected '}' in '} SLIST_H' at '}'PSLIST_HEADER;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:19252: syntax error, unexpected identifier, expec' at 'ListHead' in '      PSLIST_HEADER ListHead
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:19299: syntax error, unexpected identifier, expec' at 'ListHead' in '      PSLIST_HEADER ListHead
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:19446: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:19525: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21810: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21844: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21854: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21866: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21876: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21885: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21894: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21904: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21928: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21938: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21987: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:21997: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winnt.h:22008: syntax error, unexpected identifier in '__' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\minwinbase.h:380: syntax error, unexpected identifier, ex' at 'LPCONTEXT';' in 'typedef PCONTEXT LPCONTEXT;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\fileapi.h:933: syntax error, unexpected identifier in '  ' at 'FILE_SEGMENT_ELEMENT'egmentArray[],
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\fileapi.h:935: syntax error, unexpected typedef-name, exp' at 'LPDWORD'ier or '(' or '*' in '      LPDWORD lpReserved,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\fileapi.h:936: syntax error, unexpected typedef-name, exp' at 'LPOVERLAPPED'r '(' or '*' in '      LPOVERLAPPED lpOverlapped
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\fileapi.h:1128: syntax error, unexpected identifier in ' ' at 'FILE_SEGMENT_ELEMENT'SegmentArray[],
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\fileapi.h:1130: syntax error, unexpected typedef-name, ex' at 'LPDWORD'fier or '(' or '*' in '      LPDWORD lpReserved,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\fileapi.h:1131: syntax error, unexpected typedef-name, ex' at 'LPOVERLAPPED'or '(' or '*' in '      LPOVERLAPPED lpOverlapped
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\errhandlingapi.h:214: syntax error, unexpected identifier' at 'PCONTEXT'TEXT pContextRecord,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\interlockedapi.h:48: syntax error, unexpected identifier,' at 'ListHead'or ',' in '      PSLIST_HEADER ListHead
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\interlockedapi.h:101: syntax error, unexpected identifier' at 'ListHead' or ',' in '      PSLIST_HEADER ListHead
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\processthreadsapi.h:751: syntax error, unexpected identif' at 'LPCONTEXT'CONTEXT lpContext
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\processthreadsapi.h:781: syntax error, unexpected '*', ex' at '*'')' or ',' in '      const CONTEXT* lpContext
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winbase.h:668: syntax error, unexpected identifier, expec' at 'LPEXCEPTION_POINTERS' PEXCEPTION_POINTERS LPEXCEPTION_POINTERS;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\winerror.h:28504: syntax error, unexpected identifier' at '__forceinline'\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.17134.0\\shared\\winerror.h"
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winbase.h:8995: syntax error, unexpected identifier, expe' at 'Destination'n '      PCONTEXT Destination,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winbase.h:8997: syntax error, unexpected identifier, expe' at 'Source'';' in '      PCONTEXT Source
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winbase.h:9013: syntax error, unexpected identifier in ' ' at 'PCONTEXT'Context,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winbase.h:9036: syntax error, unexpected identifier, expe' at 'Context',' in '      PCONTEXT Context,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winbase.h:9045: syntax error, unexpected identifier, expe' at 'Context',' in '      PCONTEXT Context,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winbase.h:9047: syntax error, unexpected typedef-name, ex' at 'PDWORD'ifier or '(' or '*' in '      PDWORD Length
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winbase.h:9061: syntax error, unexpected identifier, expe' at 'Context',' in '      PCONTEXT Context,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\wingdi.h:1028: syntax error, unexpected '*', expecting ',' at '*' in 'typedef struct tagMETARECORD __unaligned *PMETARECORD;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\wingdi.h:1036: syntax error, unexpected '*', expecting ',' at '*' in 'typedef struct tagMETARECORD __unaligned  *LPMETARECORD;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\wingdi.h:1064: syntax error, unexpected '*', expecting ',' at '*' in 'typedef struct tagMETAHEADER __unaligned *PMETAHEADER;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\wingdi.h:1065: syntax error, unexpected '*', expecting ',' at '*' in 'typedef struct tagMETAHEADER __unaligned  *LPMETAHEADER;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\wingdi.h:4780: syntax error, unexpected identifier in '  ' at 'LPMETARECORD'                         LPMETARECORD lpMR,
' at ';'am Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winuser.h:5182: syntax error, unexpected ';' in ';
' at ';'am Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winuser.h:5205: syntax error, unexpected ';' in ';
c:\program files (x86)\windows kits\10\include\10.0.17134.0\shared\rpcndr.h:586: syntax error, unexpected typedef-name, ' at 'PFORMAT_STRING'or '(' in 'typedef const unsigned char  * PFORMAT_STRING;
c:\program files (x86)\windows kits\10\include\10.0.17134.0\shared\rpcndr.h:809: syntax error, unexpected ',', expecting' at ','ier or '(' or '*' or ';' in 'typedef  struct _MIDL_STUB_MESSAGE MIDL_STUB_MESSAGE,   *PMIDL_STUB_MESSAGE;
c:\program files (x86)\windows kits\10\include\10.0.17134.0\shared\rpcndr.h:1178: syntax error, unexpected ',', expectin' at ','fier or '(' or '*' or ';' in '} MIDL_SYNTAX_INFO, *PMIDL_SYNTAX_INFO;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4456: syntax error, unexpected identifier in '' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4468: syntax error, unexpected identifier in '' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4480: syntax error, unexpected identifier in '' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4491: syntax error, unexpected identifier in '' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4509: syntax error, unexpected typedef-name in '        Bytes  = (((Bytes) + ((__alignof(DEVICE_DSM_RANGE)) - 1)) / (__alignof(DEVICE_DSM_RANGE)) * (__alignof(DEVICE_' at 'DEVICE_DSM_RANGE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4533: syntax error, unexpected typedef-name in '    Bytes = (((Bytes) + ((__alignof(DEVICE_DSM_RANGE)) - 1)) / (__alignof(DEVICE_DSM_RANGE)) * (__alignof(DEVICE_DSM_R' at 'DEVICE_DSM_RANGE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4582: syntax error, unexpected identifier in '' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4612: syntax error, unexpected typedef-name in '        Bytes = (((Bytes) + ((__alignof(DEVICE_DSM_RANGE)) - 1)) / (__alignof(DEVICE_DSM_RANGE)) * (__alignof(DEVICE_D' at 'DEVICE_DSM_RANGE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4687: syntax error, unexpected typedef-name in' at 'DEVICE_DSM_RANGE'taSetRangesOffset % __alignof(DEVICE_DSM_RANGE)) {
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4744: syntax error, unexpected identifier in '' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4766: syntax error, unexpected identifier in '' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4789: syntax error, unexpected identifier in '' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winioctl.h:4818: syntax error, unexpected identifier in '' at '__forceinline'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\winscard.h:56: syntax error, unexpected typedef-name, exp' at 'LPCVOID'ier or '(' in 'typedef const void *LPCVOID;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_search.h:16: syntax error, unexpected identifie' at '__pragma'pack(push, 8))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_search.h:212: syntax error, unexpected identifi' at '__pragma'(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_wstdlib.h:479: syntax error, unexpected identif' at '__pragma'a(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\stdlib.h:1366: syntax error, unexpected identifier in '' at '__pragma'op))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\oleauto.h:1061: syntax error, unexpected typedef-name, ex' at 'LPTYPELIB'er or '(' in 'typedef /* [unique] */    ITypeLib *LPTYPELIB;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\oleauto.h:1093: syntax error, unexpected typedef-name, ex' at 'LPTYPEINFO'r or '(' in 'typedef /* [unique] */    ITypeInfo *LPTYPEINFO;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\oleauto.h:1100: syntax error, unexpected typedef-name, ex' at 'LPTYPECOMP'r or '(' in 'typedef /* [unique] */    ITypeComp *LPTYPECOMP;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\oleauto.h:1107: syntax error, unexpected typedef-name, ex' at 'LPCREATETYPELIB''(' in 'typedef ICreateTypeLib * LPCREATETYPELIB;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\oleauto.h:1109: syntax error, unexpected typedef-name, ex' at 'LPCREATETYPEINFO'(' in 'typedef ICreateTypeInfo * LPCREATETYPEINFO;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\oleauto.h:1211: syntax error, unexpected typedef-name, ex' at 'LPDISPATCH'r or '(' in 'typedef /* [unique] */    IDispatch *LPDISPATCH;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:151: syntax error, unexpected identifier i' at 'LPUWSTR'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:160: syntax error, unexpected identifier, ' at 'String1'or ',' in '      PCUWSTR String1,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:167: syntax error, unexpected identifier, ' at 'String1'or ',' in '      PCUWSTR String1,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:174: syntax error, unexpected identifier, ' at 'String' or ',' in '      LPCUWSTR String
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:194: syntax error, unexpected identifier, ' at 'String1'or ',' in '      PCUWSTR String1,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:201: syntax error, unexpected identifier, ' at 'String' or ',' in '      PCUWSTR String
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:223: syntax error, unexpected identifier, ' at 'String' or ',' in '      LPUWSTR String
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:239: syntax error, unexpected identifier, ' at 'String1'or ',' in '      LPCUWSTR String1,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:256: syntax error, unexpected identifier, ' at 'String1'or ',' in '      LPCUWSTR String1,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:273: syntax error, unexpected identifier, ' at 'String' or ',' in '      LPCUWSTR String
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:316: syntax error, unexpected '*', expecti' at '*'r ';' in 'typedef  WCHAR __unaligned *PUWSTR_C;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:329: syntax error, unexpected identifier, ' at 'String' or ',' in '      PCUWSTR String,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:344: syntax error, unexpected identifier, ' at 'String' or ',' in '      PCUWSTR String,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:420: syntax error, unexpected identifier, ' at 'Destination',' in '      PUWSTR  Destination,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:444: syntax error, unexpected identifier, ' at 'Destination',' in '      PUWSTR Destination,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:446: syntax error, unexpected identifier, ' at 'Source' or ';' in '      PCUWSTR Source
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:462: syntax error, unexpected identifier, ' at 'String' or ',' in '      PCUWSTR String
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared\stralign.h:478: syntax error, unexpected identifier, ' at 'String1'or ',' in '      PCUWSTR String1,
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um\imm.h:45: syntax error, unexpected typedef-name, expectin' at 'LPUINT'or '(' in 'typedef UINT  *LPUINT;
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_stdio_config.h:12: syntax error, unexpected ide' at '__pragma'ragma(pack(push, 8))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_stdio_config.h:122: syntax error, unexpected id' at '__pragma'pragma(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\corecrt_wstdio.h:2198: syntax error, unexpected identif' at '__pragma'a(pack(pop))
C:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt\stdio.h:2471: syntax error, unexpected identifier in '_' at '__pragma'p))
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\bin\HostX64\x64\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG -IC:\include\cairo -IC:\gtk-build\gtk\x64\release\include\glib-2.0 -IC:\gtk-build\gtk\x64\release\lib\glib-2.0\include /TcC:\gtk-build\build\x64\release\pango\win32\tmp-introspectbujlzql2\PangoCairo-1.0.c /FoC:\gtk-build\build\x64\release\pango\win32\tmp-introspectbujlzql2\PangoCairo-1.0.obj -wd4996 /IC:/gtk-build/gtk/x64/release/include/glib-2.0 /IC:/gtk-build/gtk/x64/release/lib/glib-2.0/include /IC://include/cairo
PangoCairo-1.0.c
g-ir-scanner: link: link.exe -out:C:\gtk-build\build\x64\release\pango\win32\tmp-introspectbujlzql2\PangoCairo-1.0.exe C:\gtk-build\build\x64\release\pango\win32\tmp-introspectbujlzql2\PangoCairo-1.0.obj pango-1.0.lib pangocairo-1.0.lib gio-2.0.lib gmodule-2.0.lib gobject-2.0.lib glib-2.0.lib intl.lib /libpath:C://lib cairo.lib
Microsoft (R) Incremental Linker Version 14.14.26430.0
Copyright (C) Microsoft Corporation.  All rights reserved.

Command '['C:\\gtk-build\\build\\x64\\release\\pango\\win32\\tmp-introspectbujlzql2\\PangoCairo-1.0.exe', '--introspect-dump=C:\\gtk-build\\build\\x64\\release\\pango\\win32\\tmp-introspectbujlzql2\\functions.txt,C:\\gtk-build\\build\\x64\\release\\pango\\win32\\tmp-introspectbujlzql2\\dump.xml']' returned non-zero exit status 3221225785.
NMAKE : fatal error U1077: 'C:\Users\Tim\Anaconda3\python.exe' : return code '0x1'
Stop.
Traceback (most recent call last):
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\utils\builder.py", line 361, in build
    if self.__build_one(p):
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\utils\builder.py", line 477, in __build_one
    skip_deps = proj.build()
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\projects.py", line 512, in build
    self.make_single_gir('pango')
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\projects.py", line 490, in make_single_gir
    self.exec_vs(cmd)
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\utils\base_project.py", line 82, in exec_vs
    self.builder.exec_vs(cmd, working_dir=self._get_working_dir(), add_path=add_path)
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\utils\builder.py", line 705, in exec_vs
    self.__execute(self.__sub_vars(cmd), working_dir=working_dir, add_path=add_path, env=self.vs_env)
  File "C:\Users\Tim\GitClones\gvsbuild\gvsbuild\utils\builder.py", line 737, in __execute
    subprocess.check_call(args, cwd=working_dir, env=env, shell=True)
  File "C:\Users\Tim\Anaconda3\lib\subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'nmake -f pango-introspection-msvc.mak CFG=release PREFIX=C:\gtk-build\gtk\x64\release PYTHON=C:\Users\Tim\Anaconda3\python.exe install-introspection' returned non-zero exit status 2.
Error: gobject-introspection build failed

```

I'm going to have to tackle this when I get back from vacation.

**Comment by tjwilli58** [comment]
Success! I uninstalled Anaconda completely, then installed the latest Anaconda (`Anaconda3-5.2.0-Windows-x86_64.exe`). After uninstalling libiconv, gtk3-full successfully built.

Thanks for all the help.  Only problem now is, I still can't seem to use pygobject, which was the point of all of this.

```
(base) C:\Users\Tim>python
Python 3.6.5 |Anaconda, Inc.| (default, Mar 29 2018, 13:32:41) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import gi
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Tim\Anaconda3\lib\site-packages\pygobject-3.28.1-py3.6-win-amd64.egg\gi\__init__.py", line 42, in <module>
    from . import _gi
ImportError: DLL load failed: The specified module could not be found.
>>> Anaconda3-5.2.0-Windows-x86_64.exeAnaconda3-5.2.0-Windows-x86_64.ex
KeyboardInterrupt
>>>

```
This belongs in a separate issue. (Next week sometime).

**Comment by guruDanny67** [comment]
Try to add the bin directory where you build everything
(c:\gtk-build\gtk\x64\release\bin or c:\gtk-build\gtk\win32\release\bin) to
your path before starting python.

I'm confident everything will work correctly :)

Good luck

2018-06-23 22:31 GMT+02:00 Tim Williams <notifications@github.com>:

**Comment by tjwilli58** [comment]
It turns out that if I invoke python from a regular cmd  or powershell and not an Anaconda prompt. I can do an `import gi`. 
Anaconda prompt shell:

```
(gvsbuild) C:\Users\Tim>python
Python 3.6.6 |Anaconda, Inc.| (default, Jun 28 2018, 11:27:44) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.environ['path']
'C:\\Users\\Tim\\Anaconda3\\envs\\gvsbuild;C:\\Users\\Tim\\Anaconda3\\envs\\gvsbuild\\Library\\mingw-w64\\bin;C:\\Users\\Tim\\Anaconda3\\envs\\gvsbuild\\Library\\usr\\bin;C:\\Users\\Tim\\Anaconda3\\envs\\gvsbuild\\Library\\bin;C:\\Users\\Tim\\Anaconda3\\envs\\gvsbuild\\Scripts;C:\\Users\\Tim\\Anaconda3\\envs\\gvsbuild\\bin;C:\\Users\\Tim\\Anaconda3;C:\\Users\\Tim\\Anaconda3\\Library\\mingw-w64\\bin;C:\\Users\\Tim\\Anaconda3\\Library\\usr\\bin;C:\\Users\\Tim\\Anaconda3\\Library\\bin;C:\\Users\\Tim\\Anaconda3\\Scripts;C:\\Users\\Tim\\Anaconda3\\bin;C:\\ProgramData\\Oracle\\Java\\javapath;C:\\Program Files (x86)\\Intel\\iCLS Client\\;C:\\Program Files\\Intel\\iCLS Client\\;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files\\dotnet\\;C:\\WINDOWS\\System32\\OpenSSH\\;d:\\Program Files\\Git\\cmd;C:\\Program Files (x86)\\Xoreax\\IncrediBuild;C:\\Users\\Tim\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Users\\Tim\\AppData\\Local\\Programs\\Python\\Python36;C:\\gtk-build\\gtk\\x64\\release\\bin;'
>>> import gi
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Tim\Anaconda3\envs\gvsbuild\lib\site-packages\pygobject-3.28.1-py3.6-win-amd64.egg\gi\__init__.py", line 42, in <module>
    from . import _gi
ImportError: DLL load failed: The specified procedure could not be found.

```

Powershell:

```
PS C:\Users\Tim> .\Anaconda3\envs\gvsbuild\python.exe
Python 3.6.6 |Anaconda, Inc.| (default, Jun 28 2018, 11:27:44) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.environ['path']
'C:\\Users\\Tim\\Anaconda3\\envs\\gvsbuild\\Library\\bin;C:\\ProgramData\\Oracle\\Java\\javapath;C:\\Program Files (x86)\\Intel\\iCLS Client\\;C:\\Program Files\\Intel\\iCLS Client\\;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files\\dotnet\\;C:\\WINDOWS\\System32\\OpenSSH\\;d:\\Program Files\\Git\\cmd;C:\\Program Files (x86)\\Xoreax\\IncrediBuild;C:\\Users\\Tim\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Users\\Tim\\AppData\\Local\\Programs\\Python\\Python36;C:\\gtk-build\\gtk\\x64\\release\\bin;'
>>> import gi
>>>

```

**Comment by guruDanny67** [comment]
I'm not a big Anaconda expert. Can be something in the Anaconda path that
conflict with gi.

You can try to put the c:\gtk-build\... bin directory at the beginning of
the path and not at the end.

Or use Powershell / cmd :)


2018-07-02 2:41 GMT+02:00 Tim Williams <notifications@github.com>:
